.github/workflows/validate-licenses.yaml

```diff
@@ -41,22 +42,20 @@ jobs:
           GITHUB_TOKEN: "${{ secrets.MERGEBOT_TOKEN }}"
       - name: Import GPG key
         if: |
-          steps.updateLicenses.outcome == 'success'
-          && contains(github.event.pull_request.labels.*.name, 'update-licenses')
+          contains(github.event.pull_request.labels.*.name, 'update-licenses')
         uses: crazy-max/ghaction-import-gpg@v4
         with:
           gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
           git_tag_gpgsign: true
       - name: Commit and push changes
         if: |
-          steps.updateLicenses.outcome == 'success'
-          && contains(github.event.pull_request.labels.*.name, 'update-licenses')
+          contains(github.event.pull_request.labels.*.name, 'update-licenses')
         run: |
           git config user.email ci-mergebot@d2iq.com
           git config user.name d2iq-mergebot
           git remote set-url origin https://x-access-token:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git
           git config --global url."https://${GITHUB_TOKEN}:x-oauth-basic@github.com/".insteadOf "https://github.com/"
-          git add licenses.d2iq.yaml
+          git add .
```

Comment:
@cwyl02 originally I switched to git add . because the original was creating an untracked files error, even with the edits to .gitignore, but I tested it with the original git add licenses.d2iq.yaml and it didn't produce the same error! so this is resolved.